app/controllers/user.js

```diff
@@ -1,11 +1,13 @@
 const User = require('../models/User')
 const jwt = require('jsonwebtoken')
+const emailServices = require('../services/email')
 
 module.exports = {
   createUser: async (req, res, next) => {
     const user = new User(req.body)
     try {
       await user.save()
+      emailServices.sendSuccessfulSignupEmail(req.body.email, req.body.name)
```

Comment:
Please use the updated backend code. Check the user schema once, the name is an Object

Reply:
That's why CI is failing.

Reply:
Thanks @Rupeshiya. Didn't realize it was a object. Maybe we should change the content at the documentation as well as it's misleading. Already asked @devesh-verma about the key.